features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.infer.json

```
@@ -315,6 +315,9 @@
       "identifier_first.preserve_case_sensitivity.enabled": false,
       "application_mgt.enable_cross_tenant_authorized_api_validation": false,
       "oauth.callback.restrict_fragment_components": false,
+      "oauth.jwt_x5t_s256_enabled": false,
+      "oauth.jwt_x5t_enabled": true,
+      "identity_mgt.challenge_questions.skip_sending_questions_in_url": false,
```

Biome (2.1.2)

[error] The key identity_mgt.challenge_questions.skip_sending_questions_in_url was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)